catalog/dags/providers/provider_api_scripts/nypl.py

```diff
@@ -138,7 +138,7 @@ def get_record_data(self, data):
                 "filetype": filetype,
                 "category": category,
                 "meta_data": metadata,
-                "raw_tags": NyplDataIngester._get_tags(mods) or None,
+                "raw_tags": self._get_tags(mods) or None,
```

Comment:
```suggestion
                "raw_tags": self._get_tags(mods),
```
`_get_tags` already returns set or None.